### DISCUSSION TITLE
When using metro for the web (expo router) how would I apply postcss/tailwind/nativewind

### DISCUSSION THREAD

**Discussion by dannyhw** [nativewind/nativewind#396] [Q&A]
So when nativewind for the web I know that you need to setup tailwind like you would for any other web application. However I'm trying to use expo router which instructs me to enable metro as the web bundler.

So I'm wondering if anyone has already tried this and what would be the alternative for using postcss loader on metro and are there any examples.

I know this is super specific and maybe not exactly nativewind's problem but I would really appreciate any insight on this.

Thanks.

**Comment by marklawlor** [comment]
[Metro for Web does not support CSS stylesheets](https://docs.expo.dev/guides/customizing-metro/#expo-webpack-vs-expo-metro). The best solution is to use the Tailwind CLI and manually insert the style tag into your index.html.

**Reply by dannyhw** [reply]
Interesting I'll have to experiment with that. Thanks!

**Reply by jeltehomminga** [reply]
If I follow that link it shows in the table that Metro will support the features related to CSS from expo SDK49, any plans to start supporting Metro then? Or will it then automagically start working as the same way it does now for webpack setup?

**Reply by dannyhw** [reply]
Yeah it seems like you can already enable experimental support for css in metro. 

All you have to do is import the css file somewhere.

And if you didnt see Mark joined expo and will be helping to bring css support to metro web. 

https://github.com/marklawlor/nativewind/issues/464

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
